resources/sdk/purecloudjavascript/templates/ApiClient.mustache

```diff
@@ -1,6 +1,7 @@
 import { default as axios } from 'axios';
 import Configuration from './configuration.js';
 import { default as qs } from 'qs';
+const { HttpsProxyAgent } = require('hpagent')
```

Comment:
I couldn't get the import to work like the other imports, anybody know how I could?

Reply:
Thanks Tim. `import { HttpsProxyAgent } from 'hpagent'` doesn't seem to work so I'll have to leave it as is